doc/ref_model/chapters/chapter04.md

```diff
@@ -268,60 +268,157 @@ Table 4-10 shows performance measurement capabilities.
+| SmartNIC enabled | network-smartnic | ❌ | ✅ | labels a host/node that includes a Programmable accelerator for vSwitch/vRouter, Network Function and/or Hardware Infrastructure. |  |
+| SmartSwitch enabled | network-smartswitch | ❌ | ✅ | labels a host/node that is connected to a Programmable Switch Fabric or TOR switch |  |
+
+
+<a name="4.2.4"></a>
+### 4.2.4 Workload Flavours
+
+Workload Flavours (sometimes also referred to as “compute flavours”) are sizing specifications beyond the capabilities specified by node profiles. Workload flavours represent the compute, memory, storage, and network resource sizing templates used in requesting resources on a host that is conformant with the profiles and profile extensions. The workload flavour specifies the requested resource’s (VM, container) compute, memory and storage characteristics. Workload Flavours can also specify different storage resources such as ephemeral storage, swap disk, network speed, and storage IOPs.
+
+
+
+<a name="4.2.4.2"></a>
+#### 4.2.4.1 Workload Flavours Geometry (Sizing)
+
+Workload Flavour geometry consists of the following cloud infrastructure resource elements:
+
+<a name="Table4-12"></a>
+
+| Element | Mnemonic | Description |
+|-----|----|-----|
+| cpu | c | Number of virtual compute resources (vCPUs) |
+| memory | r | Virtual resource instance memory in megabytes. |
+| storage - ephemeral | e | Specifies the size of an ephemeral/local data disk that exists only for the life of the instance. Default value is 0. <br>The ephemeral disk may be partitioned into boot (base image) and swap space disks. |
+| storage - permanent | d | Specifies the disk size of permanent storage |
+
+<p align="center"><b>Table 4-12:</b> Workload Flavour Geometry Specification.</p>
+
+The flavours syntax consists of specifying using the <element, value> pairs separated by a colon (“:”). For example, the flavour specification: {cpu : 4; memory: 8 Gi; storage-permanent: 80Gi}.
+
+<a name="4.2.4.2"></a>
+#### 4.2.4.2 Workload Flavours Extra Specifications
+
+In addition to the sizing information, a workload may need to specifiy additional capabilities. These include capabilities for workload placement such as latency, workload affinity and non-affinity. It also includes capabuilities such as workload placement on multiple NUMA nodes. The extra specifications also include the [Virtual Network Interface Specifications](#4.2.5) and [Storage Extensions](#4.2.6).
+
+| Attribute | Description |
+|----------|---------------------------|
+| CPU Allocation Ratio | Specifies the cpu allocation (a.k.a. oversubsrciption) ratio |
+| Compute Intensive |	For very demanding workloads with stringent memory access requirements, where the single NUMA bandwidth maybe a bandwidth. The Compute Intensive workload profile is used so that the workload can be spread across all NUMA nodes |
```

Comment:
"bandwidth maybe a bandwidth" -> "bandwidth maybe a limitation"?

Reply:
@petorre Thanks
```suggestion
| Compute Intensive |	For very demanding workloads with stringent memory access requirements, where the single NUMA bandwidth maybe a limitation. The Compute Intensive workload profile is used so that the workload can be spread across all NUMA nodes |
```